src/allencell_ml_segmenter/services/prediction_service.py

```diff
@@ -175,6 +176,11 @@ def build_overrides(self, checkpoint: Path) -> dict[str, Any]:
         else:
             overrides["trainer.accelerator"] = "cpu"
 
+        # pytorch needs num_workers = 0 for mac osx
+        # keeps prediction on main thread
+        if platform.system() == "Darwin":
```

Comment:
Why do we need this in 2 places?  Does prediction not use `cyto_overrides_manager` ?